src/Profiles/Shifting/DataPages/Base/ANTPLUS_ShiftingBaseMainDataPageMsg.h

```diff
@@ -0,0 +1,25 @@
+#ifndef ANTPLUS_SHIFTINGBASEMAINDATAPAGEMSG_h
+#define ANTPLUS_SHIFTINGBASEMAINDATAPAGEMSG_h
+
+#include <BaseClasses/ANTPLUS_BaseDataPageMsg.h>
+#include <Profiles/Shifting/ANTPLUS_ShiftingPrivateDefines.h>
+
+#include "ANT.h"
+
+class ShiftingBaseMainDataPageMsg : public BaseDataPageMsg<BroadcastDataMsg> {
+public:
+    ShiftingBaseMainDataPageMsg(uint8_t dataPageNumber = ANTPLUS_SHIFTING_DATAPAGE_SHIFTSYSTEMSTATUS_NUMBER);
```

Comment:
This doesn't need to take a parameter since we only have one page :)

Reply:
Parameter removed